Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -135,4 +135,33 @@ Describe "BqJob-Query" {
     }
 }
 
+Describe "Stop-BqJob" {
+
+    BeforeAll {
+        $r = Get-Random
+        $datasetName = "pshell_testing_$r"
+        $test_set = New-BqDataset $datasetName
+        $folder = Get-Location
+        $folder = $folder.ToString()
+        $filename = "$folder\classics_large.csv"
+        $table = New-BqTable -Dataset $test_Set "table_$r"
+        New-BqSchema -Name "Title" -Type "STRING" | New-BqSchema -Name "Author" -Type "STRING" |
+            New-BqSchema -Name "Year" -Type "INTEGER" | Set-BqSchema $table | 
+            Add-BqTabledata $filename CSV -SkipLeadingRows 1
+    }
+
+    It "should query out of a pre-loaded table" {
+        $job = Start-BqJob -Query "select * from book_data.classics where Year > 1900"
+        $res = $job | Stop-Bqjob
+        $can = $res | Get-BqJob
+        $can.Status.State | Should Be "DONE"
```

Comment:
Probably.  I'm generally unsatisfied with this test anyway because the query is probably going to finish long before the stop request even gets there, so I'm probably not even able to test if stop works.  I think I'm just going to have to leave this test alone until I have load/export jobs, and then maybe I can find something that takes 20-30 seconds to complete there.  Then i'll be able to kill it and poll for a response.